uberfire-backend/uberfire-backend-server/src/main/java/org/uberfire/backend/server/authz/AuthorizationPolicyVfsStorage.java

```diff
@@ -149,7 +150,34 @@ public FileVisitResult visitFile(final Path file,
 
     public boolean isPolicyFile(Path p) {
         String fileName = p.getName(p.getNameCount() - 1).toString();
-        return fileName.equals("security-policy.properties") || fileName.startsWith("security-module-");
+        return fileName.equals(POLICY_FILE) || fileName.startsWith("security-module-");
+    }
+
+    @Override
+    public void deletePolicyByGroup(Group group, AuthorizationPolicy policy) {
+        ioService.startBatch(fileSystem,
+                             new CommentedOption("system",
+                                                 "Delete Policy"));
+        try (StringWriter sw = new StringWriter()){
```

Comment:
I see that lines 161 to 174 have almost the same code as the `savePolicyIntoVfs` method.
Consider extracting duplicated code into generic methods.

Reply:
removed duplicated code